### ISSUE TITLE
Can't run on Ubuntu: cannot open shared object file

### ISSUE THREAD

**Issue by s-tikhomirov** [ZcashFoundation/zec-qt-wallet#31]
On running the binary on Ubuntu 16.04 LTS I get:

```
./zec-qt-wallet: error while loading shared libraries: libpng16.so.16: cannot open shared object file: No such file or directory
```

**Comment by adityapk00** [comment]
Thanks for filing the bug. It looks like you need to install libpng16, which you can do with
`sudo apt install libpng16-16`

I thought it was included by default in Ubuntu, but maybe not. Let me look into a more permanent solution.

**Comment by s-tikhomirov** [comment]
@adityapk00 the next error I get is:

```
./zec-qt-wallet: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.25' not found (required by ./zec-qt-wallet)
```

Possibly it would be helpful to include the full list of dependencies in the Readme.

**Comment by adityapk00** [comment]
The problem is Ubuntu 16.04 doesn't ship with the version of GLIBC needed by the C++14 compiler I'm using. Easiest way is to probably upgrade to a newer OS, but I'm trying to make it compile with an older glibc.

**Comment by adityapk00** [comment]
OK, I have a tentative build for Ubuntu 16.04 if you want to try it and let me know: 
[ubuntu1604-zec-qt-wallet-v0.2.9.tar.gz](https://github.com/adityapk00/zec-qt-wallet/files/2559403/ubuntu1604-zec-qt-wallet-v0.2.9.tar.gz)

**Comment by adityapk00** [comment]
Side Note: The problem is Ubuntu 16.04 ships with a really old version of GLIBC (2.23), which means zec-qt-wallet needs to be specifically compiled with an older glibc. 

Seems to work fine on Ubuntu 16.10 and higher

**Comment by adityapk00** [comment]
The problem was a GLIBC 2.25 dependency on getentropy(). Removed this dependency, and now it should work with older distros as well. 

Available in v0.3.2 https://t.co/0sIUACVunP
